index.html

```diff
@@ -11,14 +11,79 @@
     />
   </head>
   <body class="app-grid">
-    <section class="header"></section>
 
-    <section class="content"></section>
+    <section class="header header__bar">
+      <p>Quiz App</p>
+    </section>
+
+    <section class="content">
```

Comment:
I think this should be a `main`-element, right? 
```suggestion
    <main class="content">
```